### DISCUSSION TITLE
whisperX - "no text found"

### DISCUSSION THREAD

**Discussion by frezer2011** [SubtitleEdit/subtitleedit#6977] [Q&A]
A "no text found" error appears when using WhisperX. Please tell me how to fix the error.

**Comment by chenlung** [comment]
> A "no text found" error appears when using WhisperX. Please tell me how to fix the error.

[This](https://github.com/SubtitleEdit/subtitleedit/issues/6863#issuecomment-1561750077) might help?

**Comment by valador** [comment]
For start WhisperX need install torchvision, and for for my gtx 1080 ti not support float16 and need use float32, if start whisperx standalone - you see errors. For complete usage instruction for whisperx not full(need some tool, i dont remember, see errors)
```
conda install pytorch==2.0.0 torchvision==0.15.0 torchaudio==2.0.0 pytorch-cuda=11.8 -c pytorch -c nvidia
```
![изображение](https://github.com/SubtitleEdit/subtitleedit/assets/3517485/cdd7c0b1-f48a-4dfa-b08c-1286a5397c0e)

After use - timings no better than Whisper-Faster, even worst.

**Reply by Purfview** [reply]
> After use - timings no better than Whisper-Faster, even worst.

That's what I suspected...

> whisperx standalone

Is there such thing? Can you post a link?

**Comment by valador** [comment]
> Is there such thing? Can you post a link?

I just install WhisperX according to instruction, throw conda and pip. exe file in https://github.com/SubtitleEdit/subtitleedit/issues/7545#issuecomment-1775515571

**Reply by Purfview** [reply]
Oh, that's not "standalone" then.

**Reply by valador** [reply]
Yeah, sorry for misleading you.
